### PR TITLE
Added Products (Routes/Controllers)

### LINKED ISSUE #5: added products (controllers/routes)

### PR DESCRIPTION
**Pull request by kendrinab**
closes #5

### CODE REVIEW

**Review by kristinadurant — APPROVED** [review]

**Review by Amandat97 — APPROVED** [review]

File: server/controllers/products.js [hunk 1]
```diff
@@ -19,3 +20,67 @@ exports.createProduct = async (req, res) => {
     res.status(400).json({ error: error.message });
   }
 };
+const Product = require('../db/models/product'),
```
Comment (saritasalazar) [inline server/controllers/products.js:23]: Hey Kendra! It looks like you already have `const Product = require('../db/models/product')` and `mongoose = require('mongoose');` on lines 1 & 2.

**Review by saritasalazar — CHANGES_REQUESTED** [review]
See comment in your controller/products file. Let me know if you have any questions!

File: server/controllers/products.js (hunk 1 above)
Comment (kendrinab) [reply server/controllers/products.js]: didnt catch that! ill fix now

Comment (kendrinab) [reply server/controllers/products.js]: i was removing what I had commented out, must have missed it.

**Review by kendrinab — COMMENTED** [review]
just updated 👍

**Review by saritasalazar — APPROVED** [review]

**Pull request merged**